_studio/mfx_lib/encode_hw/hevc/agnostic/g9/hevcehw_g9_ext_brc.cpp

```diff
@@ -494,7 +494,13 @@ void ExtBRC::QueryTask(const FeatureBlocks& /*blocks*/, TPushQT Push)
             break;
         case MFX_BRC_PANIC_BIG_FRAME:
             task.bSkip = true;
+#if __clang__
```

Comment:
Can you, please, create MFX_FALLTHROUGH macro in this file: https://github.com/Intel-Media-SDK/MediaSDK/blob/master/_studio/shared/include/mfx_utils.h and use it here? Implementation should be something similar to:
```
#if __cplusplus > 201703L
#define MFX_FALLTHROUGH [[fallthrough]]
#elif __clang__ 
#define MFX_FALLTHROUGH [[clang::fallthrough]]
#else 
#define MFX_FALLTHROUGH
#endif 
```

Reply:
I am going to do this